source/read/change-streams.txt

```diff
@@ -0,0 +1,268 @@
+.. _php-change-streams:
+
+====================
+Monitor Data Changes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: watch, code example
+
+Overview
+--------
+
+In this guide, you can learn how to use a **change stream** to monitor real-time
+changes to your data. A change stream is a {+mdb-server+} feature that
+allows your application to subscribe to data changes on a collection, database,
+or deployment.
+
+When using the {+php-library+}, you can call the ``watch()`` method to return an
+instance of ``MongoDB\ChangeStream``. Then, you can iterate through change events stored 
+in the ``MongoDB\ChangeStream`` instance to monitor data changes, such as updates,
```

Comment:
Unless you need to include the phrase "change events" here, consider "...iterate through the ``MongoDB\ChangeStream`` instance to..." as a more concise alternative.

I don't think of cursor objects as something that store data, so the original phrasing as odd to me. Ignore this feedback if this is the same language you're using in other language docs, though.

Reply:
changed!